### PR TITLE
Master:  Updated changes made from training

### PR DESCRIPTION
**Pull request by rplankenhorn**

### CODE REVIEW

**Comment by rplankenhorn** [comment]
@dparne Please review.

File: nao_teleop/scripts/joystick_listener.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
```
Comment (dparne) [inline nao_teleop/scripts/joystick_listener.py:3]: are you using this?

**Comment by dparne** [comment]
👍

File: nao_teleop/scripts/joystick_listener.py (hunk 1 above)
Comment (rplankenhorn) [reply nao_teleop/scripts/joystick_listener.py:3]: Nope.  I'll remove it.

**Pull request merged**